config/locales/it.yml

```diff
@@ -0,0 +1,741 @@
+# See https://github.com/svenfuchs/rails-i18n/tree/master/rails%2Flocale for starting points.
+
+# Convenzioni adottate:
+#   - tag → etichetta
+#   - story → articolo
+#   - merge/unmerge → accorpare/scorporare
+#   - ban → bannare
```

Comment:
Ban non potrebbe essere inteso anche come azione.
Se stampato su un bottone "banna" ci starebbe meglio.
Se usato in entrambi i significati forse è meglio lasciare ban

Reply:
La riga indica solo che il termine è stato italianizzato nella coniugazione, quando è sostantivo si adopera "ban".